platform/app/.recipes/OpenResty-Orthanc-Keycloak/dockerfile

```diff
@@ -23,20 +23,35 @@
 
 
 # Stage 1: Build the application
-FROM node:11.2.0-slim as builder
+FROM node:16.15.0-slim as builder
 
 RUN mkdir /usr/src/app
 WORKDIR /usr/src/app
 
 ENV APP_CONFIG=config/docker_openresty-orthanc-keycloak.js
 ENV PATH /usr/src/app/node_modules/.bin:$PATH
 
+# Copy Files
+COPY .docker /usr/src/app/.docker
```

Comment:
is there any folder that we don't need to copy? maybe we say copy all excluding this and that?

Reply:
I think the list to exclude is bigger than the list to include.

Reply:
Also, I couldn't find a way to exclude files using the `COPY` command. But... I think I may have found a way using the `.dockerignore` file. Stay tuned.